atlas-db-controller/databaseServerResource.go

```diff
@@ -38,11 +38,6 @@ func (c *Controller) syncServer(key string) error {
 		return err
 	}
 
-	err = c.syncService(key, s)
```

Comment:
why are we removing this from here ?
line 68 will get called if line 63 succeeds.. Is that an expected behavior ?

Reply:
This is expected because there is not point in creation service if secrets is not get created.